### PR TITLE
Modified startup of application to allow for specifying perflib object names

### PR DESCRIPTION
**Pull request by jwenz723**
Here is a possible solution for allowing users to specify object names to be collected, as well as object ID's.

### CODE REVIEW

File: exporter.go [hunk 1]
```diff
@@ -182,6 +189,29 @@ func main() {
...
+		o, err := perflib.QueryPerformanceData("Global")
+		if err != nil {
+			panic(err)
+		}
+		objects = o
+	}
+
+	if perfObjectsNames != nil && len(*perfObjectsNames) > 0 {
+		loopPerfObjectsNames:
+		for _, p := range *perfObjectsNames {
```
Comment (leoluk) [inline exporter.go]: This repeated inner loop is a good candidate for refactoring into a helper method

File: exporter.go [hunk 2]
```diff
@@ -165,11 +166,17 @@ func main() {
...
 		perfObjects = kingpin.Flag(
-			"perflib.objects", "List of perflib objects to queryBuf (defaults to a built-in list)").Uint32List()
+			"perflib.objects", "List of perflib object indices to queryBuf (defaults to a built-in list)").Uint32List()
 		perfObjectsAdd = kingpin.Flag(
-			"perflib.objects.add", "List of perflib objects to add to list").Uint32List()
+			"perflib.objects.add", "List of perflib object indices to add to list").Uint32List()
 		perfObjectsRemove = kingpin.Flag(
-			"perflib.objects.remove", "List of perflib objects to remove from list").Uint32List()
+			"perflib.objects.remove", "List of perflib object indices to remove from list").Uint32List()
+		perfObjectsNames = kingpin.Flag(
```
Comment (leoluk) [inline exporter.go]: README needs to be updated to reflect the new flags. Maybe add a short explanation that either indices or names can be used to the same effect.

**Comment by jwenz723** [comment]
README.md has been updated and the repetitive loops have been refactored.

File: exporter.go (hunk 2 above)
Comment (jwenz723) [reply exporter.go]: Done

File: exporter.go (hunk 1 above)
Comment (jwenz723) [reply exporter.go]: Done

**Pull request merged**